README.md

```diff
@@ -21,15 +21,15 @@ Alfa is an open and standards-based accessibility conformance testing engine use
 
 - Alfa sets out to strike a balance between false positives and negatives with the goal of having result sets reach a high [F<sub>1</sub> score](https://en.wikipedia.org/wiki/F1_score). If a false positive is encountered, it is therefore just as important to avoid introducing a potential false negative as it is fixing the false positive.
 
-- Alfa is committed to complete transparency on how test results came to be. Every line of code that has the potential to influence test results will therefore always reside within Alfa itself and never with a third-party. This does not mean that you will never encounter a dependency from a third-party, only that there are limitations to what third-party dependencies may be used for.
+- Alfa is committed to complete transparency on how test results came to be. Every line of code that has the potential to influence test results will therefore always reside within the codebase of Alfa itself and never with a third-party. However, this does not mean that we are not using dependencies from a third-parties, only that there are limitations to how we use third-party dependencies.
```

Comment:
I mostly preferred the old version, but the "However, this does not..." is good.

Reply:
The reason why I chose to rewrite this sentence it is due to the following:
- The encounter is future tense, meaning that the developer possibly can encounter a third party dependency in the future, but not now.
- "Encounter" is a strange word since in this context in my opinion.